CMakeLists.txt

```diff
@@ -187,7 +186,12 @@ endif()
 find_package(SDL2_mixer REQUIRED)
 
 # The hint provided here is targetting Arch Linux, a distro of choice for many contributors
-find_package(yaml-cpp REQUIRED HINTS /usr/lib32/cmake/yaml-cpp)
+if ("${CMAKE_SYSTEM_NAME}" MATCHES "(Free|Net|Open|DragonFly)BSD")
+    find_package(yaml-cpp REQUIRED)
+    include_directories(${YAML_CPP_INCLUDE_DIR})
```

Comment:
This is odd. I assume the include directories were not propagated properly in previous version, but that should be addressed for all the cases, not just those select BSDs

Reply:
And where issue ?

Reply:
The issue is how you split the code, where it should not be required